packages/widget-chat/src/components/title-bar/index.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+import styles from './styles.css';
+
+import Avatar from '../avatar';
+
+class TitleBar extends Component {
+  shouldComponentUpdate(nextProps) {
+    return this.props.user !== nextProps.user;
+  }
+
+  render() {
+    const {user} = this.props;
+    const {displayName} = this.props.user;
+    return (
+      <div className={classNames(`title-bar`, styles.titleBar)}>
+        <Avatar user={user} />
+        <h2 className={classNames(`title`, styles.title)}>{displayName}</h2>
+      </div>
+    );
+  }
+}
+
+TitleBar.propTypes = {
+  user: PropTypes.object.isRequired
```

Comment:
I was passing it the full user object based off of what I saw in the avatar plugin. Although, we may only need UUID for avatar. Although, we probably don't want to do the loading of the avatar in the actual component. I'll slim this down.